controls/cis_eks.yml

```diff
@@ -0,0 +1,647 @@
+    - id: 3.1.2
+      title: >-
+        3.1.2 Ensure that the kubelet kubeconfig file ownership is set to
+        root:root
+      description: |-
+        Description:
+
+        If kubelet is running, ensure that the file ownership of its kubeconfig file is set to
+        root:root.
+
+        Rationale:
+
+        The kubeconfig file for kubelet controls various parameters for the kubelet service in the
+        worker node. You should set its file ownership to maintain the integrity of the file. The file
+        should be owned by root:root.
+      status: pending
+      rules: []
+    - id: 3.1.3
+      title: >-
+        3.1.3 Ensure that the kubelet configuration file has permissions set to 644 or more restrictive (Manual)
+      description: |-
+        Description:
+
+        Ensure that if the kubelet refers to a configuration file with the --config argument, that file
+        has permissions of 644 or more restrictive.
+
+        Rationale:
+
+        The kubelet reads various parameters, including security settings, from a config file
+        specified by the --config argument. If this file is specified you should restrict its file
+        permissions to maintain the integrity of the file. The file should be writable by only the
+        administrators on the system.
+      status: pending
+      levels:
+      - level_1
+      rules: []
+    - id: 3.1.4
+      title: >-
+        3.1.4 Ensure that the kubelet configuration file ownership is set to root:root (Manual)
+      description: |-
+        Description:
+        Ensure that if the kubelet refers to a configuration file with the --config argument, that file
+        is owned by root:root.
+
+        Rationale:
+
+        The kubelet reads various parameters, including security settings, from a config file
+        specified by the --config argument. If this file is specified you should restrict its file
+        permissions to maintain the integrity of the file. The file should be writable by only the
+        administrators on the system.
+      status: pending
+      levels:
+      - level_1
+      rules: []
+  - id: '3.2'
+    title: >-
+      3.2 Kubelet
+    description: |-
+      This section contains recommendations for kubelet configuration.
+
+      Kubelet settings may be configured using arguments on the running kubelet executable, or
+      they may be taken from a Kubelet config file. If both are specified, the executable argument
+      takes precedence.
+
+      To find the Kubelet config file, run the following command:
+
+          ps -ef | grep kubelet | grep config
+
+      If the --config argument is present, this gives the location of the Kubelet config file. This
+      config file could be in JSON or YAML format depending on your distribution.
+    status: pending
+    levels:
+    - level_2
+    controls:
+    - id: 3.2.1
+      title: >-
+        3.2.1 Ensure that the --anonymous-auth argument is set to false (Automated)
```

Comment:
Automated here doesn't mean we necessarily have an OpenSCAP control for it since the rules are empty, right?

Reply:
I think we need control according to this? https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html?highlight=automated#reporting-status

Reply:
`status: automated` means we have an OpenSCAP control for this. the CIS benchmark has `(Automated) or `(Manual)` in the title for what they consider can be automatable. We could probably get rid of these.

Reply:
Cool - yeah that sounds good. I was getting a little confused looking at the benchmark and seeing the inconsistency with what we have in this repository.